src/bpdn_model.jl

```diff
@@ -70,7 +70,7 @@ function bpdn_model(args...)
     g
   end
 
-  FirstOrderModel(obj, grad!, zero(x0), name = "BPDN"),
+  FirstOrderModel(obj, grad!, rand(Float64,size(x0)), name = "BPDN"),
```

Comment:
why change the initial guess?

Reply:
I thought it would be safer not to be on the edge of the bounds at the start, but it should work even if the initial guess is zero ! I set back the initial guess to zero.